yourgov/components/FormMobileFoodVendorPermit/utils.ts

```diff
@@ -31,3 +33,29 @@ export const yupPhoneField = yup
 			"Mobile numbers must begin with '04', landline numbers must begin with an area code",
 	})
 	.length(10, 'Phone number must be 10 digits');
+
+export function hasMultipleErrors(errors: FieldValues) {
+	return Object.keys(errors).length > 1;
+}
+
+export const formHomePage =
+	'/app/licences-and-permits/apply/mobile-food-vendor-permit/form';
+
+interface GetTaskNavigationUrlParams {
+	steps: Array<FormStep>;
+	currentStepIndex: number;
+	taskHighlight: number;
+}
+
+export function getTaskCompletionUrl({
+	currentStepIndex,
+	steps,
+	taskHighlight,
+}: GetTaskNavigationUrlParams) {
+	const nextStepUrl = steps[currentStepIndex + 1]?.href;
+	return nextStepUrl || `${formHomePage}?taskHighlight=${taskHighlight}`;
+}
+
+export function getPrevTaskKey(currentTaskKey: TaskKey) {
+	return `task${Number(currentTaskKey.replace('task', '')) - 1}` as TaskKey;
```

Comment:
This could return `task0` - is that valid?

Reply:
It can, but doesn't affect anything with the logic used in `getTaskStatus()`